src/main/webapp/app/course/plagiarism-cases/instructor-view/plagiarism-cases-instructor-view.component.html

```diff
@@ -49,20 +49,34 @@ <h5 class="mb-0">
                     </div>
                 </div>
                 <div class="card-body">
-                    @for (plagiarismCase of groupedPlagiarismCases[exercise!.id!]; track plagiarismCase) {
+                    @for (plagiarismCase of groupedPlagiarismCases[exercise!.id!]; track plagiarismCase; let j = $index) {
                         <div class="row mb-3">
                             <div class="col-1 text-center">
                                 <a [routerLink]="['/course-management', courseId, 'plagiarism-cases', plagiarismCase.id]">
-                                    {{ plagiarismCase.student.name }} ({{ plagiarismCase.student.login }})
+                                    {{ plagiarismCase.student?.name }} ({{ plagiarismCase.student?.login }})
                                 </a>
                             </div>
                             @if (plagiarismCase.plagiarismSubmissions) {
-                                <div
-                                    class="col-2 text-center"
-                                    jhiTranslate="artemisApp.plagiarism.plagiarismCases.appearsInComparisons"
-                                    [translateValues]="{ count: plagiarismCase.plagiarismSubmissions.length }"
-                                ></div>
+                                <div class="col-2 text-center">
+                                    <span
+                                        id="navigate-plagiarism-detection-{{ i }}-{{ j }}"
+                                        [routerLink]="[
+                                            '/course-management',
+                                            courseId,
+                                            getExerciseUrlSegment(plagiarismCase.exercise!.type),
+                                            plagiarismCase.exercise!.id,
+                                            'plagiarism',
+                                        ]"
+                                        class="clickable underline-hover"
+                                        [attr.aria-label]="'View plagiarism detection for ' + plagiarismCase.student?.name"
+                                    >
+                                        {{
+                                            'artemisApp.plagiarism.plagiarismCases.appearsInComparisons' | artemisTranslate: { count: plagiarismCase.plagiarismSubmissions?.length }
+                                        }}
+                                    </span>
+                                </div>
                             }
+
```

Comment:
In my opinion, it does not make sense to place the link on "Appears in x comparison(s)", as we do not link directly to the direct comparison but rather to the comparison overview. So, I would prefer to move the link to the exercise title.